### PR TITLE
Create Listening_Fitzsimons.md

### PR DESCRIPTION
**Pull request by ericfitzsimons451**
@emhickmann

### CODE REVIEW

File: 1811/Listening_Fitzsimons.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Listening: 
+
+### Introduction: 
+#### Eric Fitzsimons, ex-educator, world-traveler, excellent listener/hugger, and champion of all things inclusive.
+
+### Why this topic: 
+#### I like to joke that I was raised by women.  Nearly all of the people that were influential in shaping my worldview as I grew up were progress, liberal-minded women.  From my mother to babsitters, my youth group leader to the church pastor, many of my teachers.  When my father remarried, it was to a women's/civil-rights advocate, and it's not like he was a carbon copy of the Marlboro man.
```
Comment (emhickmann) [inline 1811/Listening_Fitzsimons.md:7]: Is it a joke that you were raised by women, or is it part of your story?  I think you could say, I like to brag that I was raised by women.  Or something like that. 

I think this personal connection is a great way to start your talk.

File: 1811/Listening_Fitzsimons.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Listening: 
+
+### Introduction: 
+#### Eric Fitzsimons, ex-educator, world-traveler, excellent listener/hugger, and champion of all things inclusive.
+
+### Why this topic: 
+#### I like to joke that I was raised by women.  Nearly all of the people that were influential in shaping my worldview as I grew up were progress, liberal-minded women.  From my mother to babsitters, my youth group leader to the church pastor, many of my teachers.  When my father remarried, it was to a women's/civil-rights advocate, and it's not like he was a carbon copy of the Marlboro man.  
+
+#### When I went to college, I was surprised by how much competition there was, even amongst my friends/roommates.  There wasn't as much celebration for each other's successes, but often jealousy.  This is the exact opposite of the growth mindset that we are hoping that more people will employ in the future.  There was too much emphasis placed on what was being said rather than what was being heard.  This created too many individuals, too much conflict.
```
Comment (emhickmann) [inline 1811/Listening_Fitzsimons.md:9]: Excellent connection and description.

File: 1811/Listening_Fitzsimons.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+#### This is not to say that all men are bad at listening, and that all women are good at it.  The important aspect here is to understand that we all have biases that can act as barriers for effective communication.
```
Comment (emhickmann) [inline 1811/Listening_Fitzsimons.md:11]: Excellent job of not doing the "all women/all men"

File: 1811/Listening_Fitzsimons.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+#### Example Barriers: 
+- Excessive Talking
```
Comment (emhickmann) [inline 1811/Listening_Fitzsimons.md:14]: I can see each of these being a quick slide.  Try not to have too much text on each slide.

File: 1811/Listening_Fitzsimons.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  - Effective listening can still happen when you disagree with someone.  In fact, this is probably a time when listening becomes more important as it will help facilitate a better outcome (all parties feel respected, ideas feel appreciated rather than in-the-way)
+- Misunderstaning
+  - Could happen because the speaker is not explaining well.
+  - Could happen because the listener is not actively trying to understand/comprehend.
+- Interrupting
+  - Indicates that your voice is more important than the person speaking.
+- Faking attention
+  - Especially in a situation where the same idea that was raised by one person, and when it is brought up again by another, that person gets credit.  The audience was clearly not actually willing to listen to this person.  
+- Bringing in Emotions
+- Fear
```
Comment (emhickmann) [inline 1811/Listening_Fitzsimons.md:28]: You might also note that listening and communicating is CULTURAL and it is different across the world.  Esp if you are in a global org, this matters.

File: 1811/Listening_Fitzsimons.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+### How does this connect to mission of Gear Up? 
+#### It emphasizes the importance of inclusivity rather than exclusivity.  It endeavors to connect people to one another rather than keeping us separated.
+
+### How will this impact you as a developer? 
+#### The more that devs are capable of hearing one another when preparing for a large project, the better the outcome.  If people don't feel that they are being heard, they will be less likely to share their ideas.  In an affirming community, even bad ideas are good, as they can flush out avenues that shouldn't be taken.
+
+### Discussion Questions
+
+#### 1. Think of one bias you have that might impact your ability to hear what a coworker might suggest as a solution to a problem.
```
Comment (emhickmann) [inline 1811/Listening_Fitzsimons.md:45]: Great questions.  You might even say, "Think of the last time you were misunderstood."  Or "think of the last time you misunderstood someone"

**Review by emhickmann — COMMENTED** [review]
@ericfitzsimons451 Excellent outline and great topic.  You have essentially determined your slides already, so put a bit of work into getting the slide outline in place and then you will be set!

**Comment by ericfitzsimons451** [comment]
Thanks for all the great feedback. I'm excited about this!

Eric

**Pull request merged**